packages/api/src/auth/github/github.mapper.ts

```diff
@@ -0,0 +1,15 @@
+import { RegisteredUser } from '@coderscamp/shared/models/user';
+
+import { GithubUser } from './github.types';
+
+export class GithubMapper {
+  static fromGithubToDomain(user: GithubUser): Omit<RegisteredUser, 'id'> {
```

Comment:
Is this valid object? Can we have "RegisteredUser" without it? If id is not set is the user already "registered"? 
Looks like we have something which can be assured by types (like every user has an id), but is not here.